x-pack/filebeat/module/aws/cloudtrail/test/update-trail-json.log-expected.json

```diff
@@ -0,0 +1,29 @@
+[
+    {
+        "@timestamp": "2016-07-14T19:15:45.000Z",
+        "aws.cloudtrail.error_code": "TrailNotFoundException",
+        "aws.cloudtrail.error_message": "Unknown trail: myTrail2 for the user: 123456789012",
+        "aws.cloudtrail.event_type": "AwsApiCall",
+        "aws.cloudtrail.event_version": "1.04",
+        "aws.cloudtrail.recipient_account_id": "123456789012",
+        "aws.cloudtrail.user_identity.access_key_id": "EXAMPLE_KEY_ID",
+        "aws.cloudtrail.user_identity.arn": "arn:aws:iam::123456789012:user/Alice",
+        "aws.cloudtrail.user_identity.type": "IAMUser",
+        "cloud.account.id": "123456789012",
+        "cloud.region": "us-east-2",
+        "event.action": "UpdateTrail",
+        "event.dataset": "aws.cloudtrail",
+        "event.id": "b7d4398e-b2f0-4faa-9c76-e2d316a8d67f",
+        "event.module": "aws",
+        "event.original": "{\"eventVersion\":\"1.04\",\"userIdentity\":{\"type\":\"IAMUser\",\"principalId\":\"EX_PRINCIPAL_ID\",\"arn\":\"arn:aws:iam::123456789012:user/Alice\",\"accountId\":\"123456789012\",\"accessKeyId\":\"EXAMPLE_KEY_ID\",\"userName\":\"Alice\"},\"eventTime\":\"2016-07-14T19:15:45Z\",\"eventSource\":\"cloudtrail.amazonaws.com\",\"eventName\":\"UpdateTrail\",\"awsRegion\":\"us-east-2\",\"sourceIPAddress\":\"205.251.233.182\",\"userAgent\":\"aws-cli/1.10.32 Python/2.7.9 Windows/7 botocore/1.4.22\",\"errorCode\":\"TrailNotFoundException\",\"errorMessage\":\"Unknown trail: myTrail2 for the user: 123456789012\",\"requestParameters\":{\"name\":\"myTrail2\"},\"responseElements\":null,\"requestID\":\"5d40662a-49f7-11e6-97e4-d9cb6ff7d6a3\",\"eventID\":\"b7d4398e-b2f0-4faa-9c76-e2d316a8d67f\",\"eventType\":\"AwsApiCall\",\"recipientAccountId\":\"123456789012\"}",
+        "event.provider": "cloudtrail.amazonaws.com",
+        "fileset.name": "cloudtrail",
+        "input.type": "log",
+        "log.offset": 0,
+        "service.type": "aws",
+        "source.address": "205.251.233.182",
+        "user.id": "EX_PRINCIPAL_ID",
+        "user.name": "Alice",
+        "user_agent.original": "aws-cli/1.10.32 Python/2.7.9 Windows/7 botocore/1.4.22"
```

Comment:
You can apply the user_agent processor to the Ingest Node configuration to parse the value.

Reply:
will do